cmd/hpecli/main.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/HewlettPackard/hpecli/pkg/ilo"
 	"github.com/HewlettPackard/hpecli/pkg/logger"
 	"github.com/HewlettPackard/hpecli/pkg/update"
+	"github.com/HewlettPackard/hpecli/pkg/cloudvolumes"
```

Comment:
General practice two separate groups (internal package and external packages) and have each group in alphabetical order